.github/workflows/deploy-prod.yml

```diff
@@ -12,6 +12,7 @@ env:
   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
   AWS_DEFAULT_REGION: eu-west-1
   REACT_APP_API_ENDPOINT: https://api.oiretutka.fi/
+  REACT_APP_DATA_ENDPOINT: https://data.oiretutka.fi/
```

Comment:
Good job remembering to update the workflow files too! :clap: 

Totally would have forgot... :sweat_smile: